assets/constants.js

```diff
@@ -578,5 +578,5 @@ var constants = {
 
   "ALLOW_YAML_FILE_UPLOAD": false,
 
-  "DEV_MODE": true
+  "DEV_MODE": true 
```

Comment:
Nit: extra space?

/cc @oppia/dev-workflow-team This seems to come up regularly, we should maybe add a check to catch this.

Reply:
Hi @seanlip There is already a lint check regarding trailing spaces. But I am not sure how it didnt get caught. (Maybe its because it is in `CONFIG_FILE_PATHS`)

\cc @apb7

Reply:
@YashJipkate thanks for the confirmation! Could we fix that? 

/cc @apb7

Reply:
/cc @YashJipkate @apb7 -- ping?

Reply:
Sorry for the delayed response here. It took me some time to figure this out but this file has an eslint disable statement on the first line [here](https://github.com/oppia/oppia/blob/develop/assets/constants.js#L1). This has been there for quite some time now (I am not sure why. @vojtechjelinek, is there any reason behind this?). I tried removing it and this file seems to have a number of violations. I have filed issue #6982 for this. Thanks!